restapi/redirects.go

```diff
@@ -0,0 +1,12 @@
+package restapi
```

Comment:
We don't need this since we can redirect directly to `/unauthorized` which is a frontend page